src/main/java/org/spongepowered/common/bridge/block/BlockBridge.java

```diff
@@ -36,33 +36,30 @@
 import org.spongepowered.api.data.value.mutable.Value;
 import org.spongepowered.api.event.CauseStackManager;
 import org.spongepowered.asm.mixin.Mixin;
-import org.spongepowered.common.interfaces.IMixinTrackable;
-import org.spongepowered.common.interfaces.world.IMixinWorldServer;
-import org.spongepowered.common.mixin.core.block.state.MixinStateImplementation;
+import org.spongepowered.common.bridge.TrackableBridge;
+import org.spongepowered.common.bridge.world.ServerWorldBridge;
 
 import java.util.List;
 import java.util.Optional;
 import java.util.function.BiConsumer;
-import java.util.function.Consumer;
 
 /**
  * A quasi interface to mix into every possible {@link Block} such that their
  * acceptable {@link BlockState}s can be created, manipulated, and applied
- * with the safety of using these instance checks of the {@link IMixinBlock}.
+ * with the safety of using these instance checks of the {@link BlockBridge}.
  * The advantage of this is that a simple cast from {@link Block} to a
- * particular {@link IMixinBlock} to take advantage of particular {@link Value}
+ * particular {@link BlockBridge} to take advantage of particular {@link Value}
  * types, are really simple to perform.
  *
  * <p>It is important to note that when using this level of implementation,
- * it is already guaranteed that a particular {@link IMixinBlock} is capable
+ * it is already guaranteed that a particular {@link BlockBridge} is capable
  * of a particular type thanks to {@link Mixin}s. All that is needed to handle
  * a particular type of {@link Value} or {@link ImmutableDataManipulator} is a
  * simple cast. This is particularly useful for {@link BlockState}s as
  * they already know the type they need to focus on.</p>
  */
-public interface IMixinBlock extends IMixinTrackable {
+public interface BlockBridge extends TrackableBridge {
```

Comment:
Do we want Bridge extending Bridges? It is a similar vein of confusion that existed when Bridges extended API.

If I have a Bridge, it should have one specific purpose.